supplementary_style_guide/style_guidelines/grammar.adoc

```diff
@@ -83,5 +83,20 @@ As needed, adjust the conversational to _fairly conversational_ for an audience
 
 Red Hat Enterprise Linux 8 delivers a stable, secure, and consistent foundation across hybrid cloud deployments with the tools needed to deliver workloads faster with less effort.
 
+== Homographs
+A homograph is a word that is spelled the same as another word but has a different meaning when used as a different part of speech.
+Using homographs close together in a sentence or paragraph might confuse readers. 
+Therefore, when possible, avoid writing sentences that use homographs close to one another. 
+
+Homographs that might commonly appear in technical documentation include the following:
+
+* attribute
+* block
+* close
+* coordinates
+* minute
+* number
+* object
+* project
```

Comment:
I do not agree to add an entry or a generic admonition to avoid the word "project". PTAL at my https://github.com/openshift/openshift-docs/pull/49201#discussion_r966092250 PR.

Reply:
Sure, suggest adding this statement: Writers should therefore be aware of this potential issue and seek alternatives if applicable and if possible, as long as it does not hinder or impact or alter the technical accuracy or interpretation.

Feel free to refine this as appropriate.

Reply:
Made an edit to emphasize this guidance as more a question of writer awareness than a hard and fast rule.